traits/category.py

```diff
@@ -88,6 +92,8 @@ def add_traits_meta_data ( self, bases, class_dict, base_traits,
 #  'Category' class:
 #-------------------------------------------------------------------------------
 
+
+@six.add_metaclass(MetaCategory)
```

Comment:
Right, I guess that's because the class decorator only has a chance to do its thing _after_ the class has been created. I haven't looked into the `six.with_metaclass` source to see what magic it uses to avoid this problem.